source/_posts/rails-docker.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploying a Rails app to a Docker container
```

Comment:
Title tag character should be 50-60 character

Reply:
Done